plugins/indexer/nexus-indexer-lucene-plugin/src/main/resources/static/js/Sonatype/repoServer/RepositoryIndexBrowserContainer.js

```diff
@@ -145,6 +145,8 @@ define('Sonatype/repoServer/RepositoryIndexBrowserContainer', function() {
 
                       var resourceURI = Sonatype.config.servicePath + '/repositories/' + options.cbPassThru.node.attributes.repositoryId + '/content' + json.data.repositoryPath;
 
+                      var payload = options.cbPassThru.container.payload;
```

Comment:
during testing I observed that for search in oss there was never a format in this particular location, but there was in a slightly different context:
options.cbPassThru.container.repositoryBrowser.payload.data.format
@dariush-griffin Is it possible that different pathways to this code store the format in a different way?